ui/src/containers/ParseUrl/index.tsx

```diff
@@ -20,7 +20,7 @@ const ParseUrl: React.FC = () => {
       };
       user.authenticate(codeFinal);
       if (user.isAuthenticated) {
-        history.goBack();
+        history('-1');
```

Comment:
can we add a comment here so that anyone looking at code can get to know what `-1` signifies